pkg/testutil/cosmosdb.go

```diff
@@ -61,6 +61,7 @@ func GetCosmosDBInstance() (string, func(), error) {
 
 	dockerPool.MaxWait = maxWait
 	log.Printf("Waiting up to %v for emulator to start", dockerPool.MaxWait)
+	// Note: this hangs for Apple chip users, and fails. See https://github.com/treeverse/lakeFS/issues/8476
```

Comment:
1. I recommend going with @N-o-Z suggestion to skip this test for Mac users by adding this `if` statement:
```
if runtime.GOOS == "darwin" {
    // skip test with message
}
```
2. I don't think this is an Apple chip issue, this is an OS issue